site/404.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Reactivesearch - UI components for Elasticsearch</title>
 	<link href="https://fonts.googleapis.com/css?family=Rubik:400,700" rel="stylesheet">
-	<link rel="stylesheet" href="styles.css">
+	<link rel="stylesheet" href="/styles.css">
```

Comment:
This could cause an issue on gh-pages since it will look at the root instead of /reactivesearch